internal/rpchelp/helpdescs_en_US.go

```diff
@@ -630,17 +630,18 @@ var helpDescsEnUS = map[string]string{
 	"ticketsforaddress--result0":  "Tickets owned by the specified address.",
 
 	// PurchaseTicketCmd help.
-	"purchaseticket--synopsis":     "Purchase ticket using available funds.",
-	"purchaseticket--result0":      "Hash of the resulting ticket",
-	"purchaseticket-spendlimit":    "Limit on the amount to spend on ticket",
-	"purchaseticket-fromaccount":   "The account to use for purchase (default=\"default\")",
-	"purchaseticket-minconf":       "Minimum number of block confirmations required",
-	"purchaseticket-ticketaddress": "Override the ticket address to which voting rights are given",
-	"purchaseticket-numtickets":    "The number of tickets to purchase",
-	"purchaseticket-pooladdress":   "The address to pay stake pool fees to",
-	"purchaseticket-poolfees":      "The amount of fees to pay to the stake pool",
-	"purchaseticket-expiry":        "Height at which the purchase tickets expire",
-	"purchaseticket-comment":       "Unused",
+	"purchaseticket--synopsis":          "Purchase ticket using available funds.",
+	"purchaseticket--result0":           "Hash of the resulting ticket",
+	"purchaseticket-spendlimit":         "Limit on the amount to spend on ticket",
+	"purchaseticket-fromaccount":        "The account to use for purchase (default=\"default\")",
+	"purchaseticket-minconf":            "Minimum number of block confirmations required",
+	"purchaseticket-ticketaddress":      "Override the ticket address to which voting rights are given",
+	"purchaseticket-numtickets":         "The number of tickets to purchase",
+	"purchaseticket-pooladdress":        "The address to pay stake pool fees to",
+	"purchaseticket-poolfees":           "The amount of fees to pay to the stake pool",
+	"purchaseticket-expiry":             "Height at which the purchase tickets expire",
+	"purchaseticket-nosplittransaction": "Whether the tx is not a split transaction",
```

Comment:
This isn't super clear.  Instead i would say something like "use ticket purchase change outputs instead of a split transaction"